operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -38,7 +38,6 @@ kiali_defaults:
     ingress_enabled: true
     namespace: "istio-system"
     secret_name: "kiali"
-    service_type: "NodePort"
```

Comment:
> If we are sure we do not want to define this and rely on the cluster system default (are we really sure we want this?)

I would reverse the burden of proof here, because it seems natural to use the system default _unless_ there's good reasons not to do so.

Reply:
I agree that system defaults are fine.
I did the requested changes.